hack/update-template-almalinux.sh

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+# Functions in this script assume error handling with 'set -e'.
+# To ensure 'set -e' works correctly:
+# - Use 'set +e' before assignments and '$(set -e; <function>)' to capture output without exiting on errors.
+# - Avoid calling functions directly in conditions to prevent disabling 'set -e'.
+# - Use 'shopt -s inherit_errexit' (Bash 4.4+) to avoid repeated 'set -e' in all '$(...)'.
+shopt -s inherit_errexit || error_exit "inherit_errexit not supported. Please use bash 4.4 or later."
+
+function almalinux_print_help() {
+	cat <<HELP
+$(basename "${BASH_SOURCE[0]}"): Update the AlmaLinux image location in the specified templates
+
+Usage:
+  $(basename "${BASH_SOURCE[0]}") [--major-version <major version>] <template.yaml>...
+
+Description:
+  This script updates the AlmaLinux image location in the specified templates.
+  If the image location in the template contains a minor version and release date in the URL,
+  the script replaces it with the latest available minor version and date.
+
+  Image location basename format:
+
+	AlmaLinux-<major version>-GenericCloud-[latest|<major version>.<minor version>-<date>].<arch>.qcow2
+
+  Published AlmaLinux image information is fetched from the following URLs:
+
+    https://repo.almalinux.org/almalinux/<major version>/cloud/<arch>/images/
+
+  To parsing html, this script requires 'htmlq' or 'pup' command.      
+  The downloaded files will be cached in the Lima cache directory.
+
+Examples:
+  Update the AlmaLinux image location in templates/**.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") templates/**.yaml
+
+  Update the AlmaLinux image location in ~/.lima/almalinux/lima.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") ~/.lima/almalinux/lima.yaml
+  $ limactl factory-reset almalinux
+
+  Update the AlmaLinux image location to major version 9 in ~/.lima/almalinux/lima.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") --major-version 9 ~/.lima/almalinux/lima.yaml
+  $ limactl factory-reset almalinux
+
+Flags:
+  --major-version <version>     Use the specified version. The version must be 8 or later.
```

Comment:
```suggestion
  --version-major <version>     Use the specified version. The version must be 8 or later.
```

So that all the version flags have the common `--version-*` prefix

Reply:
changed `--major-version` to `--version-major`